src/components/checkbox/index.tsx

```diff
@@ -174,13 +177,15 @@ class Checkbox extends Component<CheckboxPropTypes, CheckboxState> {
   render() {
     const {selectedIcon, color, iconColor, disabled, testID, label, labelStyle, style, ...others} = this.props;
     return (
-      <TouchableOpacity row onPress={this.onPress}>
-        <Animated.View
+      <View row style={style}>
+        {/*@ts-ignore*/}
+        <TouchableOpacity
           {...this.getAccessibilityProps()}
           activeOpacity={1}
           testID={testID}
           {...others}
-          style={[this.getBorderStyle(), style]}
+          style={this.getBorderStyle()}
```

Comment:
Notice that before this line was 
```
style={[this.getBorderStyle(), style]}
```

Moving the style to the top container might be a breaking change in some cases. 
For instance if I rendered a checkbox like this: 
```
<Checkbox style={{borderWidth: 3}} ..../>
```
The style now will apply on the whole container instead of only the checkbox and the user will get different results. 

In this case, we can keep passing the style to the actual checkbox and add a new prop called `containerStyle` for the whole thing. (checkbox+label)